### ISSUE TITLE
Account Settings: Notifications page headings lack contrast

### ISSUE THREAD

**Issue by sixhours** [Automattic/wp-calypso#35100]
<img width="689" alt="screen-shot-2019-07-08-at-12 57 34-pm" src="https://user-images.githubusercontent.com/2124984/62387482-b7e3f500-b528-11e9-9ef6-b2e70ad46193.png">

<img width="747" alt="Screen Shot 2019-08-02 at 1 24 10 PM" src="https://user-images.githubusercontent.com/2124984/62387557-e366df80-b528-11e9-9555-b69d02f490c6.png">

#### Steps to reproduce
1. Starting at URL: http://wordpress.com/me/notifications/
2. Look for areas of low contrast on this page; headings on the columns for each site, as noted in the above screenshots
3. We can probably swap out the existing color values for `--color-text-subtle` to pass WCAG AA standards
